File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Banco.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Banco  implements Serializable {
    private int idbanco ;
    private String descripcion;
    private String estado;
    private int id_base;

    public Banco() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Banco(int idbanco, String descripcion, String estado) {
        this.idbanco = idbanco;
        this.descripcion = descripcion;
        this.estado = estado;
    }

    public int getIdbanco() {
        return idbanco;
    }

    public void setIdbanco(int idbanco) {
        this.idbanco = idbanco;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }
}

File: webservices/CORE/Banco.php
<?php


class Banco
{
    public $idbanco,$descripcion,$estado;

    /**
     * Banco constructor.
     * @param $idbanco
     * @param $descripcion
     * @param $estado
     */
    public function __construct($idbanco, $descripcion, $estado)
    {
        $this->idbanco = $idbanco;
        $this->descripcion = $descripcion;
        $this->estado = $estado;
    }

    /**
     * @return mixed
     */
    public function getIdbanco()
    {
        return $this->idbanco;
    }

    /**
     * @param mixed $idbanco
     */
    public function setIdbanco($idbanco)
    {
        $this->idbanco = $idbanco;
    }

    /**
     * @return mixed
     */
    public function getDescripcion()
    {
        return $this->descripcion;
    }

    /**
     * @param mixed $descripcion
     */
    public function setDescripcion($descripcion)
    {
        $this->descripcion = $descripcion;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }


}